src/Components/MobileDrawer.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react'
+import {makeStyles} from '@mui/styles'
+import Box from '@mui/material/Box'
+import SwipeableDrawer from '@mui/material/SwipeableDrawer'
+import {styled} from '@mui/material/styles'
+
+
+/**
+ * @param {Object} content React component to be wrapped
+ * @return {Object} React component
+ */
+export default function MobileDrawer({content}) {
+  const [open, setOpen] = useState(false)
+  const toggleDrawer = () => setOpen(!open)
+  const classes = useStyles({isOpen: open})
+  return (
+    <div className={classes.swipeDrawer}>
+      <SwipeableDrawer
+        anchor='bottom'
+        variant='persistent'
+        open={open}
+        onClose={toggleDrawer}
+        onOpen={toggleDrawer}
+        swipeAreaWidth={drawerBleeding}
+        disableSwipeToOpen={false}>
+        <StyledBox className={classes.contentContainer}>
+          <Puller onClick={toggleDrawer}/>
```

Comment:
As we discusses - replace with up/down arrows.
https://www.figma.com/file/KDQmXLUP8nmw0RjxXRBBW2/BLDRS-assets?node-id=762%3A9

Reply:
My net can't take figma right now :( I think this is as discussed tho... lmk!